spec/factories/seeds/user_factory.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_user, class: "User") do
+    email { Faker::Internet.unique.safe_email }
+    full_name { Faker::Name.name }
```

Comment:
Just a note for when we move those back to tests to make sure we don't have names like `O'niel` which makes tests flakey, maybe we can fix that from now?